plone/app/event/profiles/default/metadata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>15</version>
+  <version>16</version>
```

Comment:
Since we do the upgrade in plone.app.upgrade, I will revert this version back to 15. Otherwise `portal_setup` silently complains:

![Screenshot 2022-03-30 at 12 23 37](https://user-images.githubusercontent.com/210587/160811720-416d8322-1ee7-4df9-88b8-7b70ad50a14c.png)